### ISSUE TITLE
fan-out, pubsub tick messages 

### ISSUE THREAD

**Issue by yinhm** [yinhm/datafeed#6]
- fan-out support(?)
- pubsub tick message
- zeromq(?) as a better socks, multiplexing, server push
- websockets(?): fits all morden browsers

**Comment by chopchopjames** [comment]
not sure if this project is still activate, but I'm switching to zmq pub/sub from redis one, much more efficient.

**Comment by yinhm** [comment]
Yeah, I had the same experience which is a Golang project. It was a private project so I can not open source it.

This project is not under active development, since I always want to ditch Python for Golang, but blocked by numpy(no numpy in Golang).
